code/upaya/app/assets/stylesheets/application.css.scss

```diff
@@ -11,6 +11,6 @@
  * file per style scope.
  *
  *= require_self
- *= require ./main
+ *= require us_web_design_standards
+ *= require us_web_design_standards_fonts
  */
```

Comment:
The instructions in the web design standards README are wrong. The way it was before with the `@import` statements is the Right Way™ to do it. See the icam-public repo for reference: https://github.com/USCIS/icam-public/blob/dev/app/assets/stylesheets/application.css.scss